packages/mobile/locales/es-419/sendFlow7.json

```diff
@@ -7,6 +7,7 @@
   "sendOrRequest": "Enviar o solicitar",
   "recent": "Recientes",
   "contacts": "Contactos",
+  "onValora": "En Valora",
```

Comment:
We should remove the translations from here and the BR file because since we're using Crowdin now it can cause conflicts in the PR Crowdin creates